lib/ex_aws/elasticache.ex

```diff
@@ -194,13 +195,12 @@ defmodule ExAws.ElastiCache do
         path: "/",
         service: :elasticache}
   """
-  @type node_group_configuration ::
-    %{
+  @type node_group_configuration :: [
```

Comment:
Switched to a keyword list to keep it consistent with all of the other elixir AWS libraries.